README.md

```diff
@@ -17,10 +17,11 @@ to use.
 ## Prerequisites
 
 * Git
-* Node.js LTS (version 16.13.0 or higher)
+* Go ^1.20.2
+* Node.js ^18.12 or ^20.9
```

Comment:
the policy is Node LTS, whatever that may be. let's leave specific version numbers out of the README.

though consider linking from here to https://github.com/Agoric/agoric-sdk/blob/master/docs/node-version.md

Reply:
Hmm, does that make sense in a list that _is_ specific with respect to other software? I think instead it makes sense to have the README be the source of truth and docs/node-version.md to include instructions for updating it in accordance with Node.js LTS policy. We shouldn't expect someone to be able to use a Node.js version that we haven't actively vetted, even if it is LTS.

Reply:
I agree with @gibson042, there is a difference between policy, and what is actually tested. The goal of this section is also to be referenced from release notes by permalink, so we should have an explicit statement of what versions are explicitly supported and tested.

Reply:
>I think instead it makes sense to have the README be the source of truth and docs/node-version.md to include instructions for updating it in accordance with Node.js LTS policy

I'm not concerned with which doc is the source of truth, but that any doc is rather than CI or package.json engines which is the actual truth about what is supported. I'm weary of multiple sources of "truth" and the likelihood they get out so sync.

Take that input fwiw. No objections to the PR as is.